content/events/24-25/lightning_workshops.md

```diff
@@ -37,4 +37,10 @@ Er wordt plaats voorzien buiten leslokaal 3.4 voor tussen tutorials door of voor
 | Prometheus, (un)common knowledge | Francis B. |
 | Self-Hosting met docker compose | Jonas |
 
+## slides
```

Comment:
Het is beter als je na een titel een newline toevoegd.
Anders gaan sommige dingen zoals tabellen niet goed werken
```suggestion
## slides

```

Reply:
this is fixed in my last commit